### PR TITLE
add Hm17

### PR DESCRIPTION
**Pull request by MichaelN98**

### CODE REVIEW

File: HM17/page_objects/main_page.py [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+
+    __2_badges = (By.XPATH, "//div[@class='home-page__head__left__bages home-page__head__left__bages__notLogin en']")
+
+    __free_download_button_in_header = (
+    By.XPATH, "//a[@class='auth_button btn btn-discount btn-discount--success NS-button NS-button__greenblueClear header__login_btn']")
+
+    __login_header = (By.XPATH, "//a[@class='auth_button']")
+    __pop_up_login = (By.XPATH, "//h1[contains(text(),'Log in to your account')]")
+
+    def is_h1_main_page_displayed(self):
```
Comment (bohdandovbsyh) [inline HM17/page_objects/main_page.py:40]: what is h1?

File: HM17/tests/test_site.py [hunk 2]
```diff
@@ -0,0 +1,207 @@
...
+
+# def test_click_logo_in_header(open_login_page):
+#     global main_page_go
+#     login_page = open_login_page
+#     try:
+#         main_page_go = login_page.click_logo_header()     не получилось понять как обойти алерт
+#     except UnexpectedAlertPresentException:
+#         alert = login_page.driver.switch_to.alert
+#         alert.accept()
+#     assert main_page_go.is_h1_main_page_displayed(), 'H1 not displayed'
```
Comment (bohdandovbsyh) [inline HM17/tests/test_site.py:205]: you should separate TC into suites. Also, all verification is on displaying only. What if an element is displayed but has the wrong text for example? You should check different data

File: HM17/utilities/web_ui/base_page.py [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+        return self.__wait.until(EC.element_to_be_clickable(locator))
+
+    def _send_keys(self, locator, value, is_clear=True):
+        element = self._wait_until_element_located(locator)
+        if is_clear:
+            element.clear()
+        element.send_keys(value)
+
+    def _click(self, locator):
+        self._wait_until_to_be_clickable(locator).click()
```
Comment (bohdandovbsyh) [inline HM17/utilities/web_ui/base_page.py:24]: please implement more functions here and use them all around FR

File: HM17/page_objects/checker_page.py [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+    def __init__(self, driver):
+        self.driver = driver
+        self.__wait = WebDriverWait(self.driver, 20, 10)
+
+    __h1_checker_page = (
+        By.XPATH, "//h1[contains(text(),'Compare your websites in bulk with SERP scraping')]")
+
+    def is_checker_page_displayed(self):
+        is_checker_page_element = self.__wait.until(EC.visibility_of_element_located(self.__h1_checker_page))
+        return is_checker_page_element.is_displayed()
```
[the remark this answers is not in the document]
Comment (MichaelN98) [reply HM17/page_objects/checker_page.py:16]: fix

File: HM17/page_objects/forgot_password_page.py [hunk 5]
```diff
@@ -0,0 +1,16 @@
+from selenium.webdriver.common.by import By
+from selenium.webdriver.support.wait import WebDriverWait
+from selenium.webdriver.support import expected_conditions as EC
+
+
+class ForgotPassword:
+    def __init__(self, driver):
+        self.driver = driver
+        self.__wait = WebDriverWait(self.driver, 20, 10)
```
[the remark this answers is not in the document]
Comment (MichaelN98) [reply HM17/page_objects/forgot_password_page.py:9]: fix

File: HM17/page_objects/login_page.py [hunk 6]
```diff
@@ -0,0 +1,155 @@
...
+    __eye = (By.XPATH, "//i[@class='fa fa-eye-slash']")
+    __show_input_password = (By.XPATH, "//input[@type='text']")
+
+    __header_logo = (By.XPATH, "//img[@class='img-header']")
+    __multisearch = (By.XPATH, "//li[@id='searchBtn']")
+    __is_open_multisearch = (By.XPATH, "//input[@id='q']")
+
+    def is_log_password_google(self):
+        is_log_password_google_element = self.__wait.until(EC.visibility_of_element_located(self.__login_password))
+        return is_log_password_google_element.is_displayed()
```
[the remark this answers is not in the document]
Comment (MichaelN98) [reply HM17/page_objects/login_page.py:52]: fix

**Pull request merged**